packages/server/beaver_triple_service/cmd/generate_jwt.go

```diff
@@ -0,0 +1,175 @@
+package cmd
+
+import (
+	"bufio"
+	"bytes"
+	"crypto/rand"
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"path"
+	"strings"
+
+	"github.com/golang-jwt/jwt/v4"
+	"github.com/google/uuid"
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v3"
+
+	jwt_types "github.com/acompany-develop/QuickMPC/packages/server/beaver_triple_service/jwt"
+)
+
+type Options struct {
+    file_path  string
+    output_dir string
+}
+
+var (
+    o = &Options{}
```

Comment:
```suggestion
    options = &Options{}
```

Reply:
Thanks for review!
I corrected with [f0ec7c6](https://github.com/acompany-develop/QuickMPC/pull/123/commits/f0ec7c68306dce0f30f1f77f8f355c89a4f89bb1)

I would appreciate your confirmation.